src/libs/MeadeStreamReader/MeadeStreamReader.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <Arduino.h>
+
+const byte NUM_CHARS = 32;
```

Comment:
I think :XGM# might return more than 32....

Reply:
This is a receive only buffer

Reply:
Ahh, doh, of course....